src/components/Filters/FilterRow/FilterRow.tsx

```diff
@@ -89,7 +88,8 @@ const getOperatorOptions = curry((operatorValue) =>
 
 const getFieldOptions = map(normalizeOptions);
 
-const isArrayOfOptionObjects = when(isArray, pipe(head, has('value')));
+const isArrayOfOptionObjects = (value) =>
+  isArray(value) && pipe(head, has('value'))(value);
```

Comment:
@ajkl2533 Could be improved? I need to return false if argument is not array

Reply:
you can do
```
const isArrayOfOptionObjects = R.both(
 RA.isArray,
 R.pipe(R.head, R.has('value'))
)
```
According to Ramda docs
```
R.both
...
Note that this is short-circuited, meaning that the second function will not be invoked if the first returns a false-y value. ...

Reply:
`R.both` this is it! Perfect